src/EnergyPlus/SizingManager.cc

```diff
@@ -3373,6 +3360,32 @@ void GetZoneSizingInput(EnergyPlusData &state)
     }
 }
 
+void ReportTemperatureInputError(
+    EnergyPlusData &state, std::string cObjectName, int const paramNum, Real64 comparisonTemperature, bool const shouldFlagSevere, bool &ErrorsFound)
+{
+    if (state.dataIPShortCut->rNumericArgs(1) < comparisonTemperature) {
+        if (shouldFlagSevere) { // heating supply air temperature is lower than cooling supply air temperature--not allowed
```

Comment:
@RKStrand I just encountered this error with a user's file. This seems overly strict. A warning would be fine, but halting the simulation seems overly strict. The only important thing is that the heating supply temp is greater than the heating setpoint. And likewise for cooling. 
```
   ** Severe  ** Sizing:Zone="C00138" has invalid data.
   **   ~~~   ** ... incorrect Zone Heating Design Supply Air Temperature=[21.00] is less than Zone Cooling Design Supply Air Temperature=[24.00]
   **   ~~~   ** This is not allowed.  Please check and revise your input.
```

Reply:
Hmmm...I don't remember the specifics anymore as to why this was made a severe.  In looking back at the issue, it seems like you advocated making this a warning.  There are some other cases that are warnings.  I'm not sure why this was made a "severe" except that it seems illogical to have a heating supply air temperature that is less than a cooling supply air temperature.  I must not have foreseen this as a possibility.  @Myoldmopar how should we proceed here?